Bottom: 7118433546673cb18bc9b41f162093cd828e837a
Top:    e4d3d3b07b98bce82fd3562414a50cd099bb7beb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 19:05:26 +0100

Refresh of puller-c-re-write-to-use-hash-

---

diff --git a/src/puller.c b/src/puller.c
index 07870a6..30011ae 100644
--- a/src/puller.c
+++ b/src/puller.c
@@ -1068,7 +1068,11 @@ find_pairs ()
   } END_LOOP;
 
   LINE_LOOP (CURRENT); {
-    Extra *e = new_line_extra (line);
+    new_line_extra (line);
+  } END_LOOP;
+
+  LINE_LOOP (CURRENT); {
+    Extra *e = LINE2EXTRA (line);
     if (line->Point1.X >= 0)
       {
 	find_pairs_1 (line, & e->start.next, line->Point1.X, line->Point1.Y);
